src/main/java/de/retest/recheck/cli/utils/FilterUtil.java

```diff
@@ -75,4 +83,31 @@ public static boolean hasValidExcludeOption( final List<String> exclude ) {
 		return true;
 	}
 
+	public static List<Path> getFilterPaths( final Filter filterFiles ) {
```

Comment:
As I'm not familiar with what is going on with the filters: Does dies really need to take a `Filter`, or would it help to narrow it down to `CompoundFilter` (and use `instance of` to guard against this outside of this method?)